cpp/installer.cc

```diff
@@ -578,6 +578,9 @@ void InstallUBTools(const bool make_install, const OSSystemType os_system_type)
     FileUtil::MakeDirectoryOrDie("/usr/local/run");
     RegisterSystemUpdateVersion();
 
+    // Install boot notification service:
+    SystemdUtil::InstallUnit(UB_TOOLS_DIRECTORY + "/cpp/data/installer/boot_notification.service");
```

Comment:
1) This should not be related to a VuFind installation. It should rather be installed on all systems (e.g. zotaut servers)-
2) If you move it out of this function be careful, it should not be executed inside docker containers (if systemd is not available).

Reply:
Also, i am not sure if InstallUnit also enables the unit, please check!

Reply:
It is installed wherever we install ub_tools and not VuFind.  Also we only use it now if systemd is available.

Reply:
Alright, sorry, the view on github was a bit irritating.
But still => Installing this service will not automatically enable it. This might also be incorrect for the other services!

Reply:
Please have another look.